6. Developer Guides/09. Realtime-API/1. Method Calls/6. Get Subscriptions/README.md

```diff
@@ -0,0 +1,12 @@
+# Subscriptions Get
+
+TOOD: I have no returned data here. Need back up.
```

Comment:
@rodrigok Could you please help me here? What is the use for this call and what it would return?